working-groups/wg-replace-loader-with-lightning.md

```diff
@@ -0,0 +1,26 @@
+# Replace DM Loader with Lightning Working Group
+
+## Goals
+
+Replace DM Loader with Lightning.
+Issues:
+
+- [Replace Loader with Lightning](https://github.com/pingcap/dm/issues/405)
```

Comment:
```
Replace DM Loader with Lightning.

Issues:

- [Replace Loader with Lightning](https://github.com/pingcap/dm/issues/405)
```

Reply:
Please add more documents about why we need to replace DM Loader with Lightning.

Reply:
It's already explained in the issue, should we duplicate it here?

Reply:
Yeah, I think it's better to add a brief introduction here, no need to copy all the contents.